<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WaveCalc</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSlider" name="horizontalSlider_Frq">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>69</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="pageStep">
     <number>250</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_Farfield">
    <property name="geometry">
     <rect>
      <x>688</x>
      <y>69</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_Freq_EndOfNearField">
    <property name="geometry">
     <rect>
      <x>688</x>
      <y>44</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Near/Far fields border (R)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Freq_AntennaSize">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>45</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Antenna Size (D)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Freq_3">
    <property name="geometry">
     <rect>
      <x>968</x>
      <y>44</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>m</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Farfield">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>40</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Distance">
    <property name="geometry">
     <rect>
      <x>582</x>
      <y>40</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Freq_4">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>45</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>m</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_Distnce">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>70</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_SpeedOfWave">
    <property name="geometry">
     <rect>
      <x>252</x>
      <y>40</y>
      <width>50</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>343</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Freq_EndOfNearField_2">
    <property name="geometry">
     <rect>
      <x>252</x>
      <y>19</y>
      <width>82</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Speed of wave</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Freq_EndOfNearField_3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>44</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>m/s</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>961</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="rowCount">
     <number>3</number>
    </property>
    <property name="columnCount">
     <number>50</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>50</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <row/>
    <row/>
    <row/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QComboBox" name="comboBox_Fix">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>121</y>
      <width>116</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Wavelength (λ)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Border (R)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Antenna Size (D)</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_Fix">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Fix</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Hold">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Hold</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Steps">
    <property name="geometry">
     <rect>
      <x>315</x>
      <y>100</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Steps</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Hold">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>121</y>
      <width>116</width>
      <height>24</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>Wavelength (λ)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Border (R)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Antenna Size (D)</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>121</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Freq_5">
    <property name="geometry">
     <rect>
      <x>198</x>
      <y>44</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>m</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Freq">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>50</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Freq_2">
    <property name="geometry">
     <rect>
      <x>87</x>
      <y>44</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Hz</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Freq">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>19</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Frequency</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Lambda">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>50</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Freq_6">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>19</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Wavelength (λ)</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_Precentage">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>120</y>
      <width>42</width>
      <height>25</height>
     </rect>
    </property>
    <property name="value">
     <number>20</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_Steps">
    <property name="geometry">
     <rect>
      <x>315</x>
      <y>120</y>
      <width>42</width>
      <height>25</height>
     </rect>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_Steps_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>100</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1021</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionHelp"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
